src/view/components/Amount.js

```diff
@@ -1,27 +1,33 @@
-import Component from '../../Component.js';
-import generateMessages from '../../utils/generateMessages.js';
-import Inputs from '../../utils/Inputs.js';
+import Component from './Component.js';
 import { LOTTO } from '../../constants/values.js';
 import { LottoStore } from '../../domain/Lotto.js';
+import { getFields } from '../../utils/domHelper.js';
+import Validator from '../../validator/step2/index.js';
 
 export default class Amount extends Component {
-  #total;
-
-  #lottoList;
+  setEvent() {
+    this.addEvent('submit', '.lotto-store__amount-form', this.handleSubmitForm.bind(this));
+  }
 
-  setUp({ setter }) {
-    this.setter = setter;
+  template() {
+    return `
+      <label class='lotto-store__amount-label'>구입할 금액을 입력해 주세요.</label>
+      <form class='lotto-store__amount-form'>
+        <input class='lotto-store__amount-input' type='number' name='amount' placeholder='금액' min='1000'/>
+        <button class='lotto-store__purchase-btn' type='submit'>구입</button>
+      </form>
+    `;
   }
 
-  async read() {
-    const amount = await Inputs.readAmount();
+  handleSubmitForm(e) {
+    e.preventDefault();
```

Comment:
e.preventDefault(); 의 역할을 의도하고 사용하신걸까요??

Reply:
현재 리렌더링이 동기적으로 일어나는 결함이 존재합니다..
따라서 기본적으로 submit 이벤트가 발생하면 `Form submission canceled because the form is not connected` 브라우저 워닝이 발생하는데요
위와 같은 문제를 해결하기 위해 폼의 제출을 취소시키고자 e.preventDefault()를 사용하였습니다!
동기적 리렌더링의 결함을 해결하고자 노력하였지만 아직까지 성공적으로 적용시키지 못했습니다...😭